server/service/src/invoice_line/outbound_shipment_line/insert/generate.rs

```diff
@@ -79,7 +78,11 @@ fn generate_line(
         item_code,
         stock_line_id: Some(stock_line_id),
         total_before_tax,
-        total_after_tax,
+        total_after_tax: if tax.is_some() {
+            total_before_tax * (f64::from(1) + tax.unwrap() / 100.0)
+        } else {
+            total_before_tax
+        },
```

Comment:
You can avoid unwrap by doing:

```rust 
total_after_tax: match tax {
  Some(tax) => total_before_tax * (1.0 + tax / 100.0)
  None => total_before_tax
},
```
Also note how you can avoid f64::from

Worth having a helper for It? e.g. `fn totalAfterTax(total_before_tax: f64, tax: Option<f64>) -> f64`